internal/indexmigration/migration.go

```diff
@@ -36,14 +36,16 @@ func Done(paths environment.Paths) (bool, error) {
 
 // Migrate removes the index directory and then clones krew-index to the new default index path.
 func Migrate(paths environment.Paths) error {
-	isMigrated, err := Done(paths)
+	isMigrated, err := done(paths)
 	if err != nil {
 		return errors.Wrap(err, "failed to check if index migration is complete")
 	}
 	if isMigrated {
-		klog.V(2).Infoln("Already migrated")
+		klog.V(2).Infoln("Already migrated.")
```

Comment:
so imagine every time krew runs in -v≥2, this will show up in the output.
but it's missing context:

- "Already migrated" → what’s migrated
- when is it started (there's no "checking index migration" msg)